Yoast/wordpress-seo
# Rename default query params key

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Rename the default query params key.

## Test instructions

This PR can be tested by following these steps:

* Link with `YoastSEO.js`, `shortlink-util-revert-rename` branch.
* Check if the assessment links still contain extra parameters.
* Specifically check the `FunctionWordsInKeyphraseAssessment` since that was added in between. Trigger this assessment by using only function words in the keyphrase. For example: `someone was here`.

## Quality assurance


Fixes https://github.com/Yoast/YoastSEO.js/issues/1889
